src/odemis/driver/pigcs.py

```diff
@@ -2866,9 +2868,7 @@ def __init__(self, name, role, port, axes, baudrate=38400,
 
         # Special support for no address
         if len(controllers) == 1 and None in controllers:
-            master = None
-        else:
-            master = 254  # Standard value for IPAccesser with multiple controllers
+            master = None  # overwrite master default value if no address
```

Comment:
I'd suggest: "direct connection to the controller" (the rest is already explained by the comment before the `if`)